### ISSUE TITLE
No automatic updates in the integration

### ISSUE THREAD

**Issue by moorboss** [lendy007/homeassistant-skodaconnect#163]
Since yesterday, I have problems with this plugin that the information of the car doesn't seems to be updated, only after a reload of the integration, the sesors get updated. Offcourse it could be related to my installation, just wondering if others have the same issue, since other sesors of other integration get updated timely.

**Comment by MarcoGos** [comment]
I noticed the car is set to unavailable regularly (every couple of hours). Only after reloading the integration the car is back online.

**Comment by jan-jeroen** [comment]
I have the same. Integration is not updating, only after restarting Home Assistant. Seems to be related to latest update of HA Core.

**Comment by ronaldt80** [comment]
This is a duplicate of #161 - @Farfar is implementing a fix.  We will just need to wait a bit for versioning and then it will appear in HACS

**Comment by ronaldt80** [comment]
> I have the same. Integration is not updating, only after restarting Home Assistant. Seems to be related to latest update of HA Core.

You can also reload the integration manually without restarting HA or trigger an automation.

**Comment by Farfar** [comment]
Fixed in latest release.
